src/config.rs

```diff
@@ -7,12 +7,20 @@ const CONFIG_FILE: &'static str = "config.json";
 
 #[derive(Serialize, Deserialize, Clone, Debug)]
 pub struct C9000Config {
-    pub baudrate: u32
+    pub baudrate: u32,
+    pub dummy_enabled: bool,
+    pub dummy_port: String,
+    pub dummy_pa_output_level: String
```

Comment:
Eine Zahl as String zu speichern ist keine schöne Lösung. Der Wert sollte schon vom Frontend in eine Zahl konvertiert werden.